app/src/lib/shells/linux.ts

```diff
@@ -99,26 +99,23 @@ export async function getAvailableShells(): Promise<
   return shells
 }
 
-export async function launch(
-  shell: IFoundShell<Shell>,
-  path: string
-): Promise<void> {
+export function launch(shell: IFoundShell<Shell>, path: string): ChildProcess {
```

Comment:
Seems like this may be a good candidate for a switch statement.

Reply:
9a7484ea7148345bb2d2d2cadc966778ef2033f8